Lib/asyncio/timeouts.py

```diff
@@ -25,18 +25,36 @@ class _State(enum.Enum):
 
 @final
 class Timeout:
+    """Asynchronous context manager which limits the time spent inside it.
+
+    Example use:
+
+        loop = asyncio.asyncio.get_running_loop()
+        async with asyncio.Timeout(loop.time() + 1):  # Run for up to a second
```

Comment:
Oh, drats. This example looked weird to me, and I just realized that this is not the recommended way to use it -- instead, one should always call the top-level `timeout()` or `timeout_at()` functions (which are exported into the `asyncio` package namespace.

So I think we have to remove the example here and replace it with a sentence explaining the recommended use.

Sorry for making you do work and then asking you to undo it!

Reply:
No problem, creating an example was fun in and of itself:)

> So I think we have to remove the example here and replace it with a sentence explaining the recommended use.

Done